README.md

```diff
@@ -22,6 +22,9 @@ _note: autodeploy from branch is currently in progress. This docs will be update
 
 ## Usage
 
+_note: This action **will not work** with forked Pull Requests. Github does not allow secrets to be shared with forked Pull Requests. Secrets are required as part of Azure interactions. The "Checks and variables" job stops this action running
+and forked requests._
```

Comment:
Jack, I'm not sure what you mean by "forked Pull Requests." If I were less familiar with Git, I might confuse it with a dedicated branch. Is this what you mean? (Am I reading [L67](https://github.com/nginxinc/docs-actions/pull/21/files#diff-01d3bd6d4beb11b3ce4e5e8c4ad621dcfe4601b57e349b5989890db2fcd07042R67) correctly?)

```suggestion
_note: This action **will not work** with Pull Requests on forked repositories. GitHub does not allow secrets to be shared with forked Pull Requests. Secrets are required as part of Azure interactions. The "Checks and variables" job stops this action from running on forked repositories._
```